db/migrate/20231123075318_create_posts.rb

```diff
@@ -0,0 +1,14 @@
+class CreatePosts < ActiveRecord::Migration[7.1]
+  def change
+    create_table :posts do |t|
+      t.references :author, null: false, foreign_key: true
+      t.string :title
```

Comment:
- [ ] I can see that your foreign key is true but it should also be mapped to use the users' table, so it needs to create author_id and a corresponding index. I would encourage you to modify it to look like `t.references :author, foreign_key: { to_table: 'users' }`